lib/awspec/type/lambda.rb

```diff
@@ -9,7 +9,55 @@ def id
     end
 
     def timeout
-      resource_via_client.timeout
```

Comment:
This timeout methods wasn't available when running rspec yesterday (before I started hacking things around) 👍